tracing-tracy/src/tests.rs

```diff
@@ -99,7 +99,7 @@ fn span_with_fields() {
     let _enter = span.enter();
 }
 
-pub(crate) fn test() {
+pub fn test() {
```

Comment:
What’s the motivation for making these `pub`? I generally tend to prefer the rule of thumb of minimal visibility as that gives the compiler and its lints more information to work with (e.g. it can tell you about unused definitions.)

Reply:
My personal clippy config again since they're only cfg(test). Reverting.